docs/workforce-management-framework/topics/pro-Using-Keycloak-implementation.adoc

```diff
@@ -0,0 +1,87 @@
+[id='enable-keycloak-integration-{chapter}']
+= Enable Keycloak Integration
+
+The {WFM-RC-NameLong} Demo Applications allow for Authentication and Access Management using
+link:http://www.keycloak.org/index.html[Keycloak].
+The {WFM-RC-NameLong} Keycloak integration is situated inside of the Demo Applications.
+
+By default, the Keycloak Authentication and Access Management is not setup with the Demo Applications.
```

Comment:
Needs extending. Avoid using "default". You can say that demo application requires additional steps to run and use keycloak etc.